scripts/build_go.sh

```diff
@@ -168,18 +177,31 @@ if [[ "$agpl" == 1 ]]; then
 fi
 
 cgo=0
+sdk=""
+if [[ "$dylib" == 1 ]]; then
+	if [[ "$os" != "darwin" ]]; then
+		error "dylib builds are not supported on $os"
+	fi
+	cgo=1
+	cmd_path="./vpn/dylib/lib.go"
```

Comment:
Should you be specifying the individual .go here or does just specifying the dir work?

Reply:
```
$ go build -o a.so -buildmode=c-shared ./vpn/dylib/
package github.com/coder/coder/v2/vpn/dylib: build constraints exclude all Go files in /home/coder/coder/vpn/dylib
```
Not sure why it doesn't, sounds like a `c-shared` thing.